agent/engine/engine_sudo_linux_integ_test.go

```diff
@@ -389,6 +389,7 @@ func TestExecCommandAgent(t *testing.T) {
 
 	verifyExecCmdAgentExpectedMounts(t, ctx, client, testTaskId, cid, testContainerName, testExecCmdHostBinDir)
 	pidA := verifyMockExecCommandAgentIsRunning(t, client, cid)
+	verifyExecAgentRunningStateChange(t, taskEngine)
```

Comment:
are we adding stop state change in a different PR?

Reply:
I'll add this in a separate PR.